examples/cfd/04_burgers.ipynb

```diff
@@ -76,12 +82,12 @@
     "init_hat(field=u, dx=dx, dy=dy, value=2.)\n",
     "init_hat(field=v, dx=dx, dy=dy, value=2.)\n",
     "\n",
-    "plot_field(u)"
+    "plot_field(u) # Function used to plot field\n"
```

Comment:
useless comment, at this point ? I mean, this is the fourth tutorial so they must have seen/used it already?

Reply:
Yes...